Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:12
+WORKDIR /app
+COPY package*.json ./
+RUN npm ci
```

Comment:
What is `ci`? I suppose `RUN npm install` should be here

Reply:
ci - lean install.

Reply:
https://docs.npmjs.com/cli/v8/commands/npm-ci

Reply:
Agree with you, I was confused because of such an error after trying to build with `npm ci` :) We need `package-lock.json` or `yarn-lock.json` to be in the project

![image](https://user-images.githubusercontent.com/52933810/147976914-20f58ac2-472d-46b2-b344-45b4ee4779ef.png)

Reply:
Yes) And I copy this file with `COPY package*.json ./` command into container image